### PR TITLE
changes_review.js: display duration in days, hours, minutes and secs

### PR DESCRIPTION
**Pull request by flepied**

### CODE REVIEW

File: web/src/components/changes_review.js [hunk 1]
```diff
@@ -15,6 +15,20 @@ import {
 
 import { Line } from 'react-chartjs-2';
 
+function secondsToDhms(seconds) {
```
Comment (morucci) [inline web/src/components/changes_review.js]: Great, just tried an it work well ! But I think we should use an existing lib for that. Could you try https://momentjs.com/docs/#/durations/ ? There is also an humanize function that seems nice https://momentjs.com/docs/#/durations/humanize/

Comment (flepied) [reply web/src/components/changes_review.js]: ok I'll try that

Comment (flepied) [reply web/src/components/changes_review.js]: done

**Comment by morucci** [comment]
Nice thanks !

**Comment by morucci** [comment]
Sorry I've clock the wrong button and close the PR instead of merging it :(. It is merged now +1

**Pull request merged**